File: BEE 1060.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        float num;
        int count = 0;
        for(int i = 0; i < 6; i++){
            num = input.nextFloat();
            if (num >= 0){
                count += 1;
            }
        }
        System.out.printf("%d valores positivos\n", count);
    }
}

File: BEE 1060.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    float num;
    int count = 0;
    for(int i = 0; i < 6; i++){
        cin >> num;
        if (num >= 0){
            count += 1;
        }
    }
    cout << count << " valores positivos" << endl;
    return 0;
}